app/routes/__index/certificate/index.tsx

```diff
@@ -85,6 +77,25 @@ export default function CertificateIndexRoute() {
     certificate?.status === 'pending' ? 15_000 : null
   );
 
+  function formatDate(val: Date): string {
```

Comment:
Was there a reason to move this fn into the functional component?
This way this re-generates every time the component renders, and I cannot see a good reason for it. Am I missing something?

If there was no explicit reason for this change, then please move it back

Reply:
There was a duplicate, and I accidentally deleted the original